kbase-extension/static/kbase/js/widgets/function_output/modeling/KBaseFBA.FBAModel.js

```diff
@@ -331,27 +329,46 @@ function KBaseFBA_FBAModel(modeltabs) {
 
     this.CompoundTab = function (info) {
         var cpd = self.cpdhash[info.id];
+        console.log(cpd)
 		var output = [{
 			"label": "Compound",
 			"data": cpd.dispid,
-		}, {
+		},{
+            "label": "Image",
+            "data":  "<img src=http://minedatabase.mcs.anl.gov/compound_images/ModelSEED/"+cpd.id.split("_")[0]+".png style='height:300px !important;'>"

```

Comment:
The url here (or at least the root of the pattern) should be in the same config file - `src/config.json`